### ISSUE TITLE
Multiple Tests Per Class

### ISSUE THREAD

**Issue by dillinghamio** [mpociot/laravel-testtools#1]
"Add Test" so that one can make multiple tests per "session" without copy and pasting each function

**Comment by mpociot** [comment]
Yup -  and I also want to add the option to only copy the test method instead of the whole class.

**Comment by abrahamgreyson** [comment]
![ifc-television-ifc-comedy-bang-2Vp0UxNcA8TEQ](https://camo.githubusercontent.com/5bec9eb649c7b09703cb50f3fc00f37dc6d89de2/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f3256703055784e6341385445512f67697068792e676966)
